modules/react/tabs/lib/cursor/useRovingFocus.tsx

```diff
@@ -58,7 +66,9 @@ export const useRovingFocus = createHook(
     React.useEffect(() => {
       if (keyDownRef.current) {
         const item = navigation.getItem(state.cursorId);
-        document.querySelector<HTMLElement>(`#${state.id}-${getIdRef.current(item)}`)?.focus();
+        document
+          .querySelector<HTMLElement>(`[id="${state.id}-${getIdRef.current(item)}"]`)
+          ?.focus();
```

Comment:
Updated this for a better selection. In the color picker it's possible that the `id` looks like this `3324jhlhkjHG-#ffffff` Because of the section `#` query selector wasn't happy, so changed the day we select the element so it doesn't matter what's in the id string